public_html/wp-content/plugins/wc-post-types/inc/rest-api.php

```diff
@@ -26,8 +26,12 @@ function expose_public_post_meta() {
 		'single'       => true,
 	];
 
+	$default_integer = wp_parse_args( array( 'type' => 'integer' ), $meta_defaults );
+
 	// Session.
-	register_post_meta( 'wcb_session', '_wcpt_session_time', wp_parse_args( [ 'type' => 'integer' ], $meta_defaults ) );
+	register_post_meta( 'wcb_session', '_wcpt_session_time',           $default_integer );
+	register_post_meta( 'wcb_session', '_wcpt_session_length_hours',   $default_integer );
+	register_post_meta( 'wcb_session', '_wcpt_session_length_minutes', $default_integer );
```

Comment:
Could this be saved (or just exposed) as one value, `_wcpt_session_length`? This way we can avoid repeated calculations, since we'll almost always want to know the full duration, not just hours/minutes.

Reply:
🤔 IIRC there was a reason I kept them separate, but maybe that was just dictated by the UI being two separate fields. I'll see if I can remember.

Reply:
I think it was just a result of the UI. I combined them in 4aa7acb286d75469cc05253ee976fb5f38ed05f4, and that seems to work fine for the purposes of the Schedule block.